Slack.Automation/Promact.Core.Repository/ScrumRepository/ScrumBotRepository.cs

```diff
@@ -64,22 +65,16 @@ public class ScrumBotRepository : IScrumBotRepository
         {
             string replyText = string.Empty;
             var user = _slackUserDetails.GetById(UserId);
-            var channel = _slackChannelDetails.GetById(ChannelId);
+            var channel = _slackChannelRepository.GetById(ChannelId);
             string text = message;
```

Comment:
Not needed. It came to my attention while changing naming convention (mentioned below) and changed it too. Will commit it soon.